show_client/interface_transceiver_cli.go

```diff
@@ -93,6 +95,74 @@ func getInterfaceTransceiverPresence(args sdc.CmdArgs, options sdc.OptionMap) ([
 	return json.Marshal(status)
 }
 
+// Command "show interfaces transceiver eeprom"
+func getEEPROM(args sdc.CmdArgs, options sdc.OptionMap) (map[string]interface{}, error) {
+	intf := args.At(0)
+
+	var dumpDom bool
+	if v, ok := options["dom"].Bool(); ok {
+		dumpDom = v
+	}
+
+	var queries [][]string
+	queries = [][]string{
+		{"APPL_DB", "PORT_TABLE"},
+	}
+
+	portTable, err := GetMapFromQueries(queries)
+	if err != nil {
+		log.Errorf("Unable to pull data for queries %v, got err %v", queries, err)
+		return nil, err
+	}
+
+	intfEEPROM := make(map[string]interface{})
+	for iface := range portTable {
+		if intf != "" && iface != intf {
+			continue
+		}
+
+		ok, err := isValidPhysicalPort(iface)
```

Comment:
This pulls from PORT_TABLE already, I suggest we directly call frontPanelPort instead of pull PORT_TABLE again.

```
role := GetFieldValueString(portTable, iface, defaultMissingCounterValue, "role")
if !isFrontPanelPort(iface, role) { continue }
```